utils/utils.go

```diff
@@ -516,3 +516,15 @@ func ReadDockerIgnore(path string) ([]string, error) {
 	}
 	return excludes, nil
 }
+
+func PrintArray(array []string) string {
+	result := "["
+	for _, arg := range array {
+		if result != "[" {
+			result += ","
+		}
+		result += fmt.Sprintf("%q", arg)
+	}
+	result += "]"
+	return result
```

Comment:
Also, won't this do the same thing for commas now?

ie, `["echo", "hi", "you"]` will be the same as `["echo", "hi,you"]` ?